api/v1/inputs.go

```diff
@@ -69,4 +69,9 @@ type ResourceRef struct {
 	Group   string `json:"group,omitempty"`
```

Comment:
Unrelated, but should we start marking fields as required? I vaguely remember we trying it and something not working as expected? Maybe I'm misremembering.